README.md

```diff
@@ -141,35 +149,120 @@ In future iterations of the project, the Transaction Submission Service will pro
+
+```sh
+stellar-disbursement-platform db auth migrate up --tenant-id=tenant_id
+stellar-disbursement-platform db sdp migrate up --tenant-id=tenant_id
+```
+
 The tables below are used to facilitate disbursements.
 
-![core schema](./docs/images/core_schema.png)
+![core schema](./docs/images/core_db_schema.png)
 
 The tables below are used to manage user roles and organizational information.
 
-![admin schema](./docs/images/admin_schema.png)
+![admin schema](./docs/images/auth_db_schema.png)
 
 #### TSS Tables
 
+**Migration CMD**
+
+```sh
+stellar-disbursement-platform db tss migrate up
+```
+
 The tables below are shared by the transaction submission service and core service.
 
-![tss schema](./docs/images/tss_schema.png)
+![tss schema](./docs/images/tss_db_schema.png)
 
 Note that the `submitter_transactions` table is used by the TSS and will be managed by the service when moved to its own project.
 
+### Event Brokers & Background jobs
+
+The SDP can use either an Event Broker or Background jobs to handle asynchronous tasks. The choice depends on the requirements of the organization using the SDP.
+Currently, the SDP only supports Kafka as an Event Broker even though it has been designed to support other brokers through the use of interfaces.
+
+> [!NOTE]  
+> In order to avoid concurrency issues, the SDP only supports one Event Broker or Background Jobs at a time.
+
+#### Kafka
+We recommend Kafka for organizations that require high throughput and low latency. Organizations that plan on hosting multiple tenants on the SDP should consider using Kafka.
+
+**1. Topics**
+
+* `events.receiver-wallets.new_invitation`: This topic is used to send disbursement invites to recipients. *[Producer: Core, Consumer: Core]*
+* `events.payment.ready_to_pay`: This topic is used to submit payments from the Core to the TSS. *[Producer: Core, Consumer: TSS]*
+* `events.payment.payment_completed`: This topic is used to notify the Core that a payment has been completed. *[Producer: TSS, Consumer: Core]*
+
+**2. Configuration**
+
+In order to use Kafka, you need to set the following environment variables for SDP and TSS. 
+
+```sh
+  EVENT_BROKER_TYPE: "KAFKA"
+  BROKER_URLS: # comma separated list of broker urls
+  CONSUMER_GROUP_ID: # consumer group id
+  KAFKA_SECURITY_PROTOCOL: # possible values "PLAINTEXT", "SASL_SSL", "SASL_PLAINTEXT" or "SSL"
+  KAFKA_SASL_USERNAME: # username for SASL authentication. Required if KAFKA_SECURITY_PROTOCOL is "SASL_SSL" or "SASL_PLAINTEXT"
+  KAFKA_SASL_PASSWORD: # password for SASL authentication. Required if KAFKA_SECURITY_PROTOCOL is "SASL_SSL" or "SASL_PLAINTEXT"
+  KAFKA_SSL_ACCESS_KEY: # access key (keystore) in PEM format. Required if KAFKA_SECURITY_PROTOCOL is "SSL" or "SASL_SSL"
+  KAFKA_SSL_ACCESS_CERTIFICATE: # certificate in PEM format that matches the access key. Required if KAFKA_SECURITY_PROTOCOL is "SSL" or "SASL_SSL"
```

Comment:
Here is only if it's `SSL`

Reply:
addressed in 1b836460e43e6fc0fc48184e4626f9186ebec034